communities.json

```
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "xstate.svg",
+      "title": "XState",
+      "quote": "XState uses Discord to help developers use state machines, statecharts, and similar state-management technologies in their projects, as well as share updates and ideas for the future of software modeling.",
+      "inviteCode": "KCtSX7Cdjh",
```

Want to update this to use the new vanity code?
```suggestion
      "inviteCode": "xstate",
```